conda_env/cli/main_create.py

```diff
@@ -42,13 +42,9 @@ def configure_parser(sub_parsers):
         help='environment definition file (default: environment.yml)',
         default='environment.yml',
     )
-    p.add_argument(
```

Comment:
Are we positive that this isn't the cause of #254? Not sure there was proper test coverage here.